articles/openshift/openshift-faq.md

```diff
@@ -10,196 +10,220 @@ ms.date: 11/04/2019
+On Azure Red Hat OpenShift 3.11 clusters infrastructure nodes are included by default.
 
-```
-oc adm policy \
-    remove-cluster-role-from-group self-provisioner \
-    system:authenticated:oauth
-```
+## Upgrades
 
-For more information, see the OpenShift documentation on [disabling self-provisioning](https://docs.openshift.com/container-platform/3.11/admin_guide/managing_projects.html#disabling-self-provisioning).
+###  What is the general upgrade process?
 
-## Can a cluster have compute nodes across multiple Azure regions?
+Patches are applied automatically to your cluster. You do not need to take any action to receive patch upgrades on your cluster.
 
-No. All nodes in an Azure Red Hat OpenShift cluster must originate from the same Azure region.
+Running an upgrade is a safe process to run and should not disrupt cluster services. The SRE can trigger the upgrade process when new versions are available or CVEs are outstanding. Available updates are tested in a staging environment and then applied to production clusters. Following best practices helps ensure minimal to no downtime.
 
-## Are master and infrastructure nodes abstracted away as they are with Azure Kubernetes Service (AKS)?
+Planned maintenance is not prescheduled with the customer. Notifications may be sent via email if communication to the customer is required.
 
-No. All resources, including the cluster master, run in your customer subscription. These types of resources are put in a read-only resource group.
+### What is the Azure Red Hat OpenShift (ARO) maintenance process?
 
-## Is Open Service Broker for Azure (OSBA) supported?
+There are two types of maintenance for ARO: upgrades and cloud provider-initiated maintenance.
+- Upgrades include software upgrades and CVEs.
+- Cloud provider-initiated maintenance includes network, storage, and regional outages. The maintenance is dependent on the cloud provider and relies on provider-supplied updates.
 
-Yes. You can use OSBA with Azure Red Hat OpenShift. See [Open Service Broker for Azure](https://github.com/Azure/open-service-broker-azure#openshift-project-template) for more information.
+### What about emergency vs. planned maintenance windows?
 
-## I am trying to peer into a virtual network in a different subscription but getting `Failed to get vnet CIDR` error.
+We do not distinguish between the two types of maintenance. Our teams are available 24/7/365 and do not use traditional scheduled “out-of-hours” maintenance windows.
 
-In the subscription that has the virtual network, make sure to register `Microsoft.ContainerService` provider with `az provider register -n Microsoft.ContainerService --wait` 
+### How will the host operating system and OpenShift software be updated?
 
-## What is the Azure Red Hat OpenShift (ARO) maintenance process?
+The host operating systems and OpenShift software are updated through our general upgrade process.
```

Comment:
Can we link to where we spell out the general upgrade process here?

Reply:
This is still in flux. I still think we should pull all the upgrade info out into a specific guide, but right now the process needs more definition, otherwise we will create more questions than answers. I vote we keep this as is for now, but add a card to update and consolidate upgrade info once the team has nailed down the process.

Reply:
Per our conversation I updated as follows.
"The host operating systems and OpenShift software are updated as Azure Red Hat Open consumes minor release versions and patches from upstream OpenShift Container Platform."